argo/apps/templates/sbat-iam-init.yaml

```diff
@@ -0,0 +1,29 @@
+# This initializer pod will be deployed in the namespace with the environment type = 'FIDC'
+# Otherwise, currently this pod will be deployed in cdk namespace only
+# @See the pipeline initialise, step 'install_initializer_cron_job'
+
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: iam-initialiser-{{ .Release.Name }}
+  annotations:
+    argocd.argoproj.io/sync-wave: "2"
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  destination:
+    namespace: {{ .Values.spec.destination.namespace }}
+    server: {{ .Values.spec.destination.server }}
+  project: {{ .Values.spec.project }}
+  source:
+    helm:
+      parameters:
+      - name: environment.fr_platform.type
+        value: {{ .Values.environment.type }}
+      - name: iam_initializer_image_location
+        value: eu.gcr.io/sbat-gcr-develop/securebanking/secureopenbanking-uk-iam-initializer:latest
```

Comment:
Could we parametrise the tag value to be changed if necessary? with something like we use in rs helm template and other helm templates.
Example:
```yaml
- name: deployment.imageOverride.tag
        value: {{ .Values.iam-initializer.tag }}
```

Reply:
Resolved